### ISSUE TITLE
Find duplicate directories

### ISSUE THREAD

**Issue by skritland** [qarmin/czkawka#676]
It would be nice to not only find duplicate files but duplicate whole directories. If for example I have two folders with photos backup from my phone it would be nice to see if the whole folders are the same so I can just delete one or they differ (for example one is newer backup with more photos).

**Comment by m-sundman** [comment]
Yes! And mark which directory contains no other files than what's in the other, matching directory. (If both directories contain exactly the same files then both would be marked.)

**Comment by bbigras** [comment]
Yes! My main usage for a tool like this is to get rid of backups of backups.

**Comment by tmladek** [comment]
+1, this would be lovely to add!

1. I have a lot of duplicate files that on their own aren't big, but add up.
2. I have a lot of directories that are identical, but take up space predominantly because of a handful of huge files - deleting those leaves a lot of cruft on the filesystem in the form of defunct directories with tiny (but duplicate) files.

**Comment by nathanbrauer** [comment]
Just like @bbigras, I too tend to end up with a lot of backups of backups. This feature would be a lifesaver.

**Comment by AndreiMiculita** [comment]
Also agree. Would be nice if I could find which parts of my directory trees are copies of one another. My head kinda hurts trying to think of how to do that. good luck
